benchmarking/benchmark_CNOT.jl

```diff
@@ -0,0 +1,37 @@
+using BenchmarkTools
+using Snowflake
+using JSON
+using Plots
+
+include("SnowflakeBenchmarking.jl")
+
+@task "CNOT" nqubits=nqubits begin
+    map(nqubits) do k
+        t = @benchmark apply_gate!(ψ, control_x(control_qubit_1,target_qubit_1)) setup=(ψ=rand_state($k))
+        minimum(t).time
+    end
+end
+
+outputpath=joinpath(commonpath,"data_CNOT")
+
+if !ispath(outputpath)
+    mkpath(outputpath)
+end
+
+write(joinpath(outputpath,"data_CNOT_$(time_stamp).json"), JSON.json(benchmarks))
+
+plot(nqubits,
```

Comment:
How do you plot multiple benchmarks (`Yao`, `Snowflake.jl` old and new, etc.) on the same graph? It appears that it only ever plots the current benchmark.